src/jdb_to_nwb/convert_photometry.py

```diff
@@ -1,22 +1,218 @@
 from pynwb import NWBFile
+import scipy.io # for loading in signals.mat files
+import pandas as pd # essential functions
+import numpy as np # essential functions
+from scipy.sparse import diags, eye, csc_matrix  # for creating sparse matrices
+from scipy.sparse.linalg import spsolve  # for solving sparse linear systems
+from sklearn.linear_model import Lasso # for Lasso regression
 
+def WhittakerSmooth(data,binary_mask,lambda_):
```

Comment:
```suggestion
def WhittakerSmooth(data, binary_mask, lambda_):
```

Super minor formatting suggestion. It helps other readers of your code if the code follows common Python conventions for formatting. Different people have different opinions on what the conventions are and how strictly they should be followed. For an opinionated set of conventions and a script that checks for and tries to fix code to match those conventions, check out [`black`](https://github.com/psf/black). I would recommend we use `black` to avoid spending time changing code formatting or discussing it, but let's discuss together next time.

I also suggest renaming `lambda_` to just `lambda` unless there is a reason to name it that way. The underscore just raises questions.

Reply:
the name lambda doesn't work because of lambda functions! I generally use lam for variable names instead

Reply:
Oh! I forgot about that!